# Algorithm Parameters
wall_distance: 0.25
front_turn_distance: 0.35
wall_track_distance: 0.45
angle_error_range: [-0.2, 0.2]
distance_error_range: [-0.15, 0.15]
front_angle_range: [340, 20]
left_angle_range: [60, 120]
left_front_angle_range: [60, 80]
left_back_angle_range: [100, 120]

# Hardware Parameters
max_linear_velocity: 0.22
max_angular_velocity: 2.84
max_scan_range: 3.5
